packages/modular-scripts/src/program.ts

```diff
@@ -157,7 +147,7 @@ program
   .option('--updateSnapshot, -u', testOptions.updateSnapshot.description)
   .option('--verbose', testOptions.verbose.description)
   .option('--watch', testOptions.watch.description)
-  .option('--watchAll [value]', testOptions.watchAll.description, !isCI)
+  .option('--watchAll [value]', testOptions.watchAll.description, false)
```

Comment:
Setting watchAll to true for tests if not running in CI resulted in unexpected Jest behaviour for our users where jest wouldn't exit automatically after a run, so taking the opportunity to change this in modular 4